templates/form/link.tpl

```diff
@@ -7,7 +7,4 @@
  *
  * Form link control (used most commonly as a cancel link)
  *}
-
-<div{if $FBV_layoutInfo} class="{$FBV_layoutInfo}"{/if}>
-	<a href="{$FBV_href}" id="{$FBV_id}" class="{$FBV_class}">{translate key=$FBV_label}</a>
-</div>
+<a href="{$FBV_href}" id="{$FBV_id}" class="{$FBV_class}">{translate key=$FBV_label}</a>
```

Comment:
(This template doesn't appear to broadly used; I suggest we put it on the chopping block.)